core/keys/site.go

```diff
@@ -53,4 +53,7 @@ const (
 	BatteryMode   = "batteryMode"
 	BatteryPower  = "batteryPower"
 	BatterySoc    = "batterySoc"
+
+	// external battery control
+	BatteryModeExternal = "batteryModeExternal"
```

Comment:
This one I'm not sure about. What makes an external battery mode different from an internal one? I guess it should be published to `batteryMode.`

Reply:
Schau mal bitte- ich habe es jetzt so geändert, dass es überschreibbar wird. Die _Änderung_ des Wertes passiert dann aber erst im nächsten Regelintervall.